aurora/contracts/src/AuroraErc20FastBridge.sol

```diff
@@ -367,7 +368,8 @@ contract AuroraErc20FastBridge is Initializable, UUPSUpgradeable, AccessControlU
         bytes memory callbackArg = abi.encodeWithSelector(
             this.initTokenTransferCallback.selector,
             msg.sender,
-            initTransferArgs
+            initTransferArgs,
+            token
```

Comment:
Could we check somehow that the token address is consistent with initTransferArg?

Anyone could call this function and provide inconsistent data, and other applications can rely on that.

Reply:
Why anyone if it is a callback call that is restricted? 
I think even the init args should should be passed in parsed way to avoid double decoding.